database/seeders/UserSeeder.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Hash;
+
+class UserSeeder extends Seeder
+{
+	/**
+	 * Run the database seeds.
+	 *
+	 * @return void
+	 */
+	public function run()
+	{
+		DB::table("users")->insert([
+			"name" => "Petras",
+			"email" => "teacher@roctilburg.nl",
+			"password" => Hash::make("SE123!"),
+		]);
+	}
```

Comment:
You could just simply use User model or a user factory like this: `User::factory()->create([...])`